tests/QueryType/Select/RequestBuilderTest.php

```diff
@@ -78,36 +78,36 @@ public function testSelectUrlWithSortAndFilters()
     {
         $this->query->addSort('id', Query::SORT_ASC);
         $this->query->addSort('name', Query::SORT_DESC);
-        $this->query->addFilterQuery(new FilterQuery(['key' => 'f1', 'query' => 'published:true']));
+        $this->query->addFilterQuery(new FilterQuery(['local_key' => 'f1', 'query' => 'published:true']));
         $this->query->addFilterQuery(
-            new FilterQuery(['key' => 'f2', 'tag' => ['t1', 't2'], 'query' => 'category:23'])
+            new FilterQuery(['local_key' => 'f2', 'local_tag' => ['t1', 't2'], 'query' => 'category:23'])
         );
         $request = $this->builder->build($this->query);
 
         $this->assertNull($request->getRawData());
 
         $this->assertSame(
             'select?omitHeader=true&wt=json&json.nl=flat&q=*:*&start=0&rows=10&fl=*,score&sort=id asc,name desc'.
-            '&fq=published:true&fq={!tag=t1,t2}category:23',
+            '&fq={!key=f1}published:true&fq={!key=f2 tag=t1,t2}category:23',
```

Comment:
this response (along with the one in ``testSelectUrlWithCachedFilters``) has changed because of the inclusion of the ``key`` local parameter.

both are valid notations afaik, but as solarium's internals use the key for different purposes other than adding it to the query, it's not really an optional value. if adding the key as a local parameter should be optional we should move to changing the way solarium's internals stores it's queries